lms/templates/ccx/schedule.html

```diff
@@ -55,112 +44,42 @@ <h2 id="ccx_schedule_set_date_heading"></h2>
-      </div>
-      <div class="field">
-        <label for="ccx_sequential" class="form-label"><b>${_('Subsection')}</b></label>
-        <select name="sequential" id="ccx_sequential"></select>
-      </div>
-      <div class="field">
-        <label for="ccx_vertical" class="form-label"><b>${_('Unit')}</b></label>
-        <select name="vertical" id="ccx_vertical"></select>
-      </div>
-      <div class="ccx_start_date_time_fields">
-        <div class="field datepair">
-          <label for="ccx_start_date" class="form-label">
-              <b>${_('Start Date')}</b>
-              <span class="sr">
-                ## Translators: This explains to people using a screen reader how to interpret the format of YYYY-MM-DD
-                &nbsp;${_('format four digit year dash two digit month dash two digit day')}
-              </span>
-          </label>
-          <input placeholder="${_('yyyy-mm-dd')}" type="text" class="date" name="start_date" id="ccx_start_date" />
-          ## Translators: This explains to people using a screen reader how to interpret the format of HH:MM
-          <label for="ccx_start_time" class="sr form-label">${_('Start time format two digit hours colon two digit minutes')}</label>
-          <input placeholder="${_('time')}" type="text" class="time" name="start_time" id="ccx_start_time"/>
-        </div>
-      </div>
-      <div class="ccx_due_date_time_fields">
-        <div class="field datepair">
-          <label for="ccx_due_date" class="form-label">
-              <b>${_('Due Date')}</b> ${_('(Optional)')}
-              <span class="sr">
-                  ## Translators: This explains to people using a screen reader how to interpret the format of YYYY-MM-DD
-                  &nbsp;${_('format four digit year dash two digit month dash two digit day')}
-              </span>
-          </label>
-          <input placeholder="${_('yyyy-mm-dd')}" type="text" class="date" name="due_date" id="ccx_due_date"/>
-          ## Translators: This explains to people using a screen reader how to interpret the format of HH:MM
-          <label for="ccx_due_time" class="sr form-label">${_('Due Time format two digit hours colon two digit minutes')}</label>
-          <input placeholder="${_('time')}" type="text" class="time" name="due_time" id="ccx_due_time"/>
-        </div>
-      </div>
-      <div class="field">
-        <br/>
-        <button id="add-unit-button">${_('Add Unit')}</button>
-      </div>
-      <div class="field">
-        <br/>
-        <button id="add-all">${_('Add All Units')}</button>
-      </div>
-    </form>
-    <div id="all-units-added" tabindex="-1" role="region">
-      ${_("All units have been added.")}
-    </div>
-  </div>
-</div>
-
 <script>
 $(function() {
-    schedule_template = _.template($('#ccx-schedule-template').html());
-    var view = new edx.ccx.schedule.ScheduleView({
-        el: $('#new-ccx-schedule')
+    var require = window.require || RequireJS.require;
+    require([
```

Comment:
I believe this is why the cached file is not being used-- you must use the require_module static tag like in this example:

````
<%block name="js_extra">
<%static:require_module module_name="js/groups/views/cohorts_dashboard_factory" class_name="CohortsFactory">
    <%
    cohorted_user_partition = get_cohorted_user_partition(course)
    content_groups = cohorted_user_partition.groups if cohorted_user_partition else []
    %>
    var cohortUserPartitionId = ${cohorted_user_partition.id if cohorted_user_partition else None | n, dump_js_escaped_json},
        contentGroups = [
            % for content_group in content_groups:
            {
                id: ${content_group.id | n, dump_js_escaped_json},
                name: "${content_group.name | n, js_escaped_string}",
                user_partition_id: cohortUserPartitionId
            },
            % endfor
        ];
    CohortsFactory(contentGroups, "${get_studio_url(course, 'group_configurations') | n, js_escaped_string}");
</%static:require_module>
</%block>